packages/twenty-front/src/modules/object-record/record-table/record-table-cell/hooks/useTriggerActionMenuDropdown.ts

```diff
@@ -25,19 +26,21 @@ export const useTriggerActionMenuDropdown = ({
     recordTableId,
   );
 
+  const actionMenuDropdownId =
+    getActionMenuDropdownIdFromActionMenuId(actionMenuInstanceId);
+
   const recordIndexActionMenuDropdownPositionState = extractComponentState(
     recordIndexActionMenuDropdownPositionComponentState,
-    getActionMenuDropdownIdFromActionMenuId(actionMenuInstanceId),
-  );
-
-  const isActionMenuDropdownOpenState = extractComponentState(
-    isDropdownOpenComponentState,
-    getActionMenuDropdownIdFromActionMenuId(actionMenuInstanceId),
+    actionMenuDropdownId,
   );
 
   const { setActiveDropdownFocusIdAndMemorizePrevious } =
     useSetActiveDropdownFocusIdAndMemorizePrevious();
 
+  const { openDropdown } = useDropdownV2();
```

Comment:
Shouldn't we use useDropdown instead and remove the call to set active dropdown focus id ?